client/faucet/src/mint.rs

```diff
@@ -134,7 +134,7 @@ impl Service {
             seq,
             1_000_000,
             0,
-            LBR_NAME.to_owned(),
+            TMPCOIN_NAME.to_owned(),
```

Comment:
why we change LBR to TMPCOIN?

Reply:
Since minting of LBR is no longer permitted allowed after this PR the "default coin" is switching from LBR to TMPCOIN